lib/machines.js

```diff
@@ -279,6 +278,10 @@ function translate(machine, req) {
         });
     }
 
+    if (machine.internal_metadata && machine.internal_metadata.encrypted) {
+        msg.encrypted = machine.internal_metadata.encrypted;
```

Comment:
Just checking - we leave out the "encrypted" property when the vm is not encrypted (i.e. this property is optional).

Reply:
Yeah, it just matter when it's encrypted. Whatever else doesn't really matter